introduction.md

```diff
@@ -0,0 +1 @@
+Introduction to mastering github course.
```

Comment:
Why am I receiving these emails?

From: Code School Kiddo [mailto:notifications@github.com]
Sent: Thursday, March 03, 2016 11:46 AM
To: deadlyvipers/dojo_rules <dojo_rules@noreply.github.com>
Subject: Re: [dojo_rules] introduction file added (#8609)


In introduction.md<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_deadlyvipers_dojo-5Frules_pull_8609-23discussion-5Fr54908087&d=CwMCaQ&c=7MSSWy9Bs2yocjNQzurxOQ&r=SaMWGD57Rx24YE8TMMsoKw&m=hbkvR5lkbTW3uWNLGiHtLuw6ddD9r3TkMi-tz4A6lJ4&s=ZfniQsHEC2J3-4iDwY7HTasdR0RFcl-7jEhCk_PBpvo&e=>:

> @@ -0,0 +1 @@

> +Introduction to mastering github course.

Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

—
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_deadlyvipers_dojo-5Frules_pull_8609_files-23r54908087&d=CwMCaQ&c=7MSSWy9Bs2yocjNQzurxOQ&r=SaMWGD57Rx24YE8TMMsoKw&m=hbkvR5lkbTW3uWNLGiHtLuw6ddD9r3TkMi-tz4A6lJ4&s=1ceA8wcJHDg1zHvZjhKqdCrBJzjM0I0WVPEzF0IEthE&e=>.